### DISCUSSION TITLE
Plugin of Visual Studio is not working, permanently hangs

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51706] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have the latest VS 2022 and alongside Resharper installed. In Resharper settinge I followed the guide to enable VS Intellisense and checked the Checkbox to disable Resharper completions when Github Copilot is enabled.

I am facing a not responding VS IDE in my Xamarin.Forms solution when I hit Strg + T (Find Files Resharper command).

**Comment by MissieKahsay** [comment]
Hi @ericbrunner,

It's possible that the conflict between Resharper and Github Copilot is causing Visual Studio to become unresponsive when you use the Resharper command. Since you've already disabled Resharper completions when Github Copilot is enabled, the issue may be related to Resharper's indexing process, which can consume a lot of resources and cause performance issues.

To address the issue, you can try disabling Resharper temporarily to see if it resolves the problem. You can do this by going to Extensions > ReSharper > Suspend Now. If suspending Resharper resolves the issue, you may want to consider adjusting Resharper's indexing settings to reduce the number of resources it consumes.

You can also try resetting Visual Studio's settings to their default values, which can help resolve conflicts and performance issues. To do this, go to Tools > Import and Export Settings > Reset all settings.

**Reply by ghost** [reply]
Hi @MussieTeka 

Thanks, but I want to use both Plugins. Resharper+ Github CoPilot.

**Reply by ayim** [reply]
@ericbrunner would love to see this in action, please mail me at ayim at github dot com so we can find some time to understand what you're seeing. FWIW v1.78.0.0 of the VS Copilot extension fixed an issue where Copilot was causing VS to hang on solution/file load

**Reply by ghost** [reply]
After reinstall Gitub Copilot hitting STRG + T to browser all files (Resharper command binding) I experience no VS hang anymore. I'll get back after an evaluation

**Comment by ghost** [comment, marked as answer]
@ayim I can confirm that with the latest Github CoPilot Extension that issue is solved.

**Reply by MissieKahsay** [reply]
@ericbrunner I am glad you managed to solve this one.
